index.html

```diff
@@ -111,7 +111,7 @@ <h4>A smart assistant to make driving more safe, efficient, and fun by unlocking
           <li>css</li>
```

Comment:
[ OPTIONAL ]Please add all unnecessary files and folders like `.vscode`to `.gitignore`.